InstallSpinnaker.sh

```diff
@@ -68,6 +70,8 @@ usage: $0 [--cloud_provider <aws|google|none|both>]
 
     --google_region <arg>       default region for google
     --google_zone <arg>         default zone for google
+    --google_project_id <arg>   default project for google
+    --google_creds_path  <arg>  path to google service account credentials
 
```

Comment:
I dont think these belong here.
I'd like to handle configuration as a separate task.
Especially because there are lots of providers and it is going to get complicated.
Plus I think it will be a fairly common use case to want to change this post install anyway (e.g. add another provider down the road).